templates/home/_portfolio.html.twig

```diff
@@ -1,5 +1,28 @@
 {% form_theme form 'bootstrap_4_layout.html.twig' %}
 <section class="portfolio my-5">
+    {% if columnErrors is defined %}
+        <div class="d-flex align-items-center flex-column my-5">
+            <div class="bubble">
+                <div class="bubble-text">
+                    <div class="d-flex align-items-center mx-3">
+                        <i class="fas fa-exclamation-triangle fa-3x mr-3"></i>
+                        <h3 class="mb-0">Votre fichier contient des erreurs</h3>
+                    </div>
+                    <ul class="m-3 p-0">
+                        {% for key, columnError in columnErrors %}
+                            {% if key is empty %}
+                                <li class="my-2"><i class="fas fa-caret-square-right mr-2"></i>{{ columnError }}</li>
+                            {% else %}
+                                <li class="my-2"><i class="fas fa-caret-square-right mr-2"></i><b>{{ key }} :</b> {{ columnError }}</li>
+                            {% endif %}
```

Comment:
je suis pas sur de comprendre ce qu'est la key et la column error, mais dans tous les cas c'est pas simplifiable en qq chose comme
 {% for key, columnError in columnErrors %}   
<li class="my-2"><i class="fas fa-caret-square-right mr-2"></i>{{ key is empty ? '' : key ~ ' :' }} {{ columnError }}</li> 
{%endfor%}